### PR TITLE
Use logstash redis defaults.

### LINKED ISSUE #24: Use elk defaults
In general, logsearch should probably use the same defaults as the underlying components--having different defaults is potentially confusing to operators. For example, the default logstash redis congestion threshold is zero, meaning that queue size is unlimited, but the default in this release is 1m: https://github.com/cloudfoundry-community/logsearch-boshrelease/blob/8cb7e9391027012b439246f67186fdc0d9e3310b/jobs/ingestor_syslog/spec#L65-L67. I think it would be less confusing to stick to elk defaults and let operators override them as needed--or not set defaults at all, and force operators to think about tuning those settings. What do you think?

### PR DESCRIPTION
**Pull request by jmcarp**
[Resolves #24]

Most of the diffs here are changing tabs to spaces. The only meaningful change is dropping the default redis congestion properties, then only adding them to the redis options if defined.

### CODE REVIEW

File: jobs/ingestor_syslog/templates/config/logstash.conf.erb [hunk 1]
```diff
@@ -67,35 +67,39 @@ output {
...
+            "batch" => true,
+            "batch_events" => 50,
+          }
+          if_p('redis.congestion_interval') do |interval|
+            output['options']['congestion_interval'] = interval
+          end
+          if_p('redis.congestion_threshold') do |threshold|
+            output['options']['congestion_threshold'] = threshold
+          end
+        %>
```
Comment (rogeruiz) [inline jobs/ingestor_syslog/templates/config/logstash.conf.erb:90]: 72 to 87 are the changes introduced in this PR. Most of the rest is whitespace.

**Review by Infra-Red — CHANGES_REQUESTED** [review]
Hi @jmcarp, everything looks fine, but could you please update redis-input-plugin for parser logstash instances to also use the ELK default settings.

**Comment by jmcarp** [comment]
The only non-default option I saw for redis input was `threads`, which I moved into the generic `options` hash, which I updated to be merged into the individually defined options. Let me know if I missed anything else!

**Review by Infra-Red — APPROVED** [review]

**Comment by Infra-Red** [comment]
LGTM

**Pull request merged**